lib/menu.rb

```diff
@@ -0,0 +1,14 @@
+class Menu
+
+  def initialize
+    @dishes = [Dish.new("Scrambled Eggs", 5),
+                Dish.new("Steak", 15),
+                 Dish.new("Hot Sauce", 3),
+                  Dish.new("Hash Browns", 5)]
+  end
+
+  def list_dishes
+    @dishes
+  end
+
```

Comment:
This is a nice way of storing your menu. Perhaps when you list dishes, you could format the currency so that it displays as e.g. £5.00? Might it also be useful to make the menu private so that a user can't edit the prices?